docs/source/scenarios/supply_chain.rst

```diff
@@ -0,0 +1,225 @@
+
+**System Status**
+
+In order to get known of the system status, we could find the information we need from the following two interfaces:
+
+* ``env.summary`` contains all static information about the system. For example, SKU list, unit settings, facilities settings, spatial distribution of facilities, etc..
+* ``env.snapshot_list`` contains all dynamic information that describes the system's running status. For example, production progress, inventory status, vehicles' positions, demand shortage, etc..
+
+**Action**
+
+Actions in the Supply Chain scenario are simple. There are only two types of actions: ``ConsumerAction`` and ``ManufactureAction``.
+
+``ConsumerAction`` refers to the action that one facility purchases some SKUs from another facility. The information of each ``ConsumerAction`` is listed as below:
+
+* **id** (int): The corresponding ID of the consumer. Because one facility may have multiple consumer units, **this ID is a unit ID** for unambiguity.
+* **product_id** (int): The SKU id of the product.
+* **source_id** (int): The corresponding ID of the product provider. **This ID is a facility ID rather than a unit ID**, because the consumer only cares about which facility the product is purchased from, and does not care which unit produces the product.
+* **vlt** (int): The expecting velocity of the transportation.
+* **reward_discount** (int): Literal meaning.
+
+``ManufactureAction`` refers to the instruction for the facility to start production. Its information is listed as below:
+
+* **id** (int): The corresponding ID of the production unit. For one facility, each SKU has a corresponding production unit, so the ID here is a unit ID and it inherently contains the SKU id.
+* **production_rate** (int): Speed of production.
+
+The IDs used in these two actions can be found in ``env.summary``.
+
+Once a ``ManufactureAction`` is issued, the facility/unit will **immediately** (within the same tick that the action is issued) try to start producing the product. However, the production process may be stalled due to insufficient raw materials, insufficient storage, etc.. Once a ``ConsumerAction`` is issued, the source facility will try to allocate vehicles to transport products **in the next tick**. The transportation process may also not proceed smoothly due to insufficient vehicles and insufficient destination storage, etc.. Therefore, the users need to read the static/dynamic status of the system and keep optimizing their strategy.

```

Comment:
I remember that if the storage of the destination facility is insufficient, the extra part would be lost? (But I'm not sure... I has been a long time since I read the BE code Orz

Reply:
The extra part will stay on the vehicle and wait for the next unloading attempt. I just double-checked this logic.